### DISCUSSION TITLE
Replace Gitter With Discord or Github Discussions

### DISCUSSION THREAD

**Discussion by ThePieMonster** [gitextensions/gitextensions#11635] [Ideas]
### Environment

Web

### Issue description

Since Gitter is rarely (if ever) used by maintainers here, removing the link to Gitter in the read me would benefit end users who are asking for support.

Either replace with a Discord server or just link the Github Discussion page as a additional support location. 

### Steps to reproduce

Read the Readme!

### Did this work in previous version of GitExtensions?

N/A

### Diagnostics

N/A

**Comment by RussKie** [comment]
Thank you for the suggestion, though I don't know if anyone from the Core team is spending time on Discord. (E.g., I don't, even for work, though we have some .NET related channels). I think the gitter community happened historically and organically, but you're right, it's hardly more alive than a ghost town.

I'm personally open to retiring the gitter channel and redirecting users here.

**Reply by ThePieMonster** [reply]
Ya I think removing the social platforms like Gitter / Discord and using the Discussions page here is the best choice.

**Comment by xan2622** [comment]
I second this suggestion. A Discord server would be very appreciated. 
I am surprised the best Git client doesn't have one already.

**Reply by pmiossec** [reply]
With the recent news about discord (and maybe the start of enshitification), maybe it's not the better timing to invest on it...

**Reply by RussKie** [reply]
I don't think I used gitter in the past 5 years. Our channel there hadn't seen much use anyway.

I think the only meaningful way to engage with the team is here on GitHub.
